awx/main/management/commands/create_oauth2_token.py

```diff
@@ -0,0 +1,26 @@
+from awx.api.serializers import OAuth2TokenSerializer
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+from django.core.exceptions import ObjectDoesNotExist
+
+
+class Command(BaseCommand):
+    """Command that creates an OAuth2 token for a certain user. Returns the value of created token."""
+    help='Usage: awx-manage create_oauth2_token --user=username. Will return the value of created token.'
```

Comment:
Help should just be a short summary of what the command does, e.g. "Creates an OAuth2 token for a user."

`BaseCommand`/`argparse` will already output the usage if you run `awx-manage create_oauth2_token --help`.